openedx-unsupported/configuration
# Add ECOMMERCE_SECURE_PROXY_SSL_HEADER

Description
---
This PR adds `ECOMMERCE_SECURE_PROXY_SSL_HEADER` ansible variable and the `SECURE_PROXY_SSL_HEADER` configuration on `ecommerce.yml`

Testing
---

1. Provision a VM for testing (Vagrant is recommended).
2. Run the Ecommerce playbook without any variables to make sure there's a sane default:
```
$ ansible-playbook -vvv --user=ubuntu -i hosts playbooks/ecommerce.yml
```
3. Make sure the defaults produce a sane `/edx/etc/ecommerce.yml`:
```
(...)
SECURE_PROXY_SSL_HEADER: null
(...)
```
4. Run the playbook setting a value for the added config:
```
$ ansible-playbook -vvv --user=ubuntu -i hosts playbooks/ecommerce.yml -e '{"ECOMMERCE_SECURE_PROXY_SSL_HEADER":["HTTP_X_FORWARDED_PROTO","https"]}'
```
5. Make sure the resulting configuration is correct:
```
(...)
SECURE_PROXY_SSL_HEADER:
- HTTP_X_FORWARDED_PROTO
- https
(...)
```

Configuration Pull Request
---

Make sure that the following steps are done before merging:
